### PR TITLE
script/image: cleanup

### PR DESCRIPTION
**Pull request by antonlacon**
This unifies the code style used within script/image, and clarifies some comments. Two things that came up, and not addressed here, is that there is a test for powerpc64. I'm not aware of a powerpc64 target. Was this meant to be aarch64?

The second issue is that there is a line that adds a "Kodi commit: git hash" to RELEASE_DIR/RELEASE file, by way of scripts/git_version. git_version checks PKG_BUILD/VERSION for information. Kodi, at lease in my build for RPi2, does not create a VERSION file. It has a version.txt file, but that doesn't have the git hash. Checking for VERSION files in the other packages, only about 1/6 of the built packages for the contain such a file. Is this working as intended?

The changes in projects/RPi/options clean up comments only, so tossed that in here too.

### CODE REVIEW

File: scripts/image [hunk 1]
```diff
@@ -89,24 +89,17 @@ rm -rf $STAMPS_INSTALL
...
-mkdir -p $INSTALL/proc
-mkdir -p $INSTALL/run
-mkdir -p $INSTALL/sys
-mkdir -p $INSTALL/tmp
-mkdir -p $INSTALL/usr
-mkdir -p $INSTALL/var
-mkdir -p $INSTALL/flash
-mkdir -p $INSTALL/storage
+BASELAYOUT_DIRECTORIES="etc dev proc run sys tmp usr var flash storage"
+for directory in $BASELAYOUT_DIRECTORIES; do
```
Comment (MilhouseVH) [inline scripts/image]: The variable isn't really necessary, the following would work fine:
```
for directory in etc dev proc run sys tmp usr var flash storage; do
  mkdir -p $INSTALL/$directory
done
```

**Comment by MilhouseVH** [comment]
Thanks, all looks fine - just the one small nit.

`zstd` is also a SQUASHFS_COMPRESSION option for Generic, if you wanted to add it...

File: scripts/image [hunk 2]
```diff
@@ -89,24 +89,17 @@ rm -rf $STAMPS_INSTALL
...
+  mkdir -p $INSTALL/$directory
+done
 ln -sf /var/media $INSTALL/media
 ln -sf /usr/lib $INSTALL/lib
 ln -sf /usr/bin $INSTALL/bin
 ln -sf /usr/sbin $INSTALL/sbin
 
 if [ "$TARGET_ARCH" = "x86_64" -o "$TARGET_ARCH" = "powerpc64" ]; then
```
Comment (MilhouseVH) [inline scripts/image]: Yes, we should remove `-o "$TARGET_ARCH" = "powerpc64"` as it is not supported.

Same here: https://github.com/LibreELEC/LibreELEC.tv/blob/652f675e7daef4f3ea93764ccfe1ca13517c80fb/packages/virtual/initramfs/package.mk#L27

**Comment by MilhouseVH** [comment]
> The second issue is that there is a line that adds a "Kodi commit: git hash" to RELEASE_DIR/RELEASE file,

Yes, this isn't working (I wonder if it ever did). What you currently get is a RELEASE file that contains (in a Generic tar, for example):
```
Generic.x86_64-8.2.5
Kodi commit:
```

I don't find anything in the LE repo which uses/reads the RELEASE file (unless it's used by a back-end distribution process - @chewitt?) so it's possible we could drop it entirely along with all associated code including the deletion of `scripts/git_version` (`get_pkg_version $MEDIACENTER` is a more reliable alternative anyway).

If we do need to keep the RELEASE file then we can fix the broken Kodi component by replacing the call to `scripts/git_version` with `get_pkg_version`.

**Comment by antonlacon** [comment]
Updated to address comments. Replaced `scripts/git_version` with `get_pkg_version` pending a decision on what to do with the RELEASE file.

**Comment by antonlacon** [comment]
Should `scripts/git_version` get deleted as well? This looks like its sole use.

**Comment by antonlacon** [comment]
I believe this is squared away.

**Comment by MilhouseVH** [comment]
Excellent, many thanks!

**Pull request merged**